shinobi.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "shinobi",
+  "plugin_schema": "2",
+  "release": "11.3-RELEASE",
```

Comment:
Can we please use `12.2-RELEASE` here ?

Reply:
Done. Actually changed it to 13.1-RELEASE.